app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanelList.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <div class="notification-list-item--wrap h-full flex-view ">
+    <div
+      v-for="notificationItem in notifications"
+      v-show="!isLoading"
+      :key="notificationItem.id"
+      class="notification-list--wrap flex-view "
+      @click="() => onClickNotification(notificationItem)"
```

Comment:
Make this a normal button, we have to handle Accessibility standards if we are adding click on a div.

Reply:
fixes by https://github.com/chatwoot/chatwoot/pull/4061/commits/8ae236b568fa2cd316395131f5222bb81f5743e6